### PR TITLE
Add constants

### PR DESCRIPTION
**Pull request by StevenDoesStuffs**
Fixes https://github.com/DefinitelyTyped/DefinitelyTyped/issues/9573

### CODE REVIEW

File: mathjs/mathjs.d.ts [hunk 1]
```diff
@@ -3,6 +3,11 @@
 // Definitions by: Ilya Shestakov <https://github.com/siavol/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+// Type definitions for mathjs
+// Project: http://mathjs.org/
+// Definitions by: Ilya Shestakov <https://github.com/siavol/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```
Comment (vvakame) [inline mathjs/mathjs.d.ts]: duplicated?

**Comment by mhegazy** [comment]
@StevenTheEVILZ can you please remove the duplicate header?

**Comment by StevenTheEVILZ** [comment]
Sorry, I'm new to git. How do I do that?

**Comment by StevenTheEVILZ** [comment]
Starting new pull request: see 9921

**Pull request closed without merging**